pkg/deployment/deployChart.go

```diff
@@ -0,0 +1,39 @@
+package deployment
+
+import (
+        "gopkg.in/ini.v1"
+        "github.com/thkukuk/kubic-control/pkg/tools"
+)
+
+func DeployChart(chartName, releaseName, valuesPath string) (bool, string) {
+
+	var success bool
+	var message string	
+	if valuesPath == ""{
+		success, message = tools.ExecuteCmd("helm", "install", releaseName,
+			chartName, "--kubeconfig=/etc/kubernetes/admin.conf")
+	}else{
+		success, message = tools.ExecuteCmd("helm", "install", releaseName,
+			chartName, "--kubeconfig=/etc/kubernetes/admin.conf",
+			"-f", valuesPath)
+	}
+	
+	if success != true {
+		return success, message
+	}
+
+	result, err := tools.Sha256sum_f(chartName)
+
+	cfg, err := ini.LooseLoad("/var/lib/kubic-control/k8s-yaml.conf")
```

Comment:
If you add this to k8s-yaml, kubicd will use kubectl to update the helm chart, I'm pretty sure this will fail.
In the end we need, beside for kubectl and kustomize, a third implementation for installing and updating helm charts.

Reply:
I kinda saw that coming. Thanks for the feedback. I'll try to look at the code more in details. 

How do you usually test kubicd ?